sources/manticore-worker/replica.php

```diff
@@ -94,6 +94,13 @@ function checkIsJoinNodeReady($joinHost, $qlPort, $clusterName)
     Logger::info("One pod");
     $manticoreJson->startManticore();
     $manticore = new ManticoreConnector('localhost', $qlPort, $clusterName, -1);
+
+    $settings = $manticore->showSettings();
+    if (!isset($settings['searchd.data_dir'])){
+        Logger::error("Data dir option wasn't passed in searchd conf. Replication disabled");
```

Comment:
"Data dir" -> data_dir

"searchd conf" - replace with balancer.config.content to make it more clear?